### DISCUSSION TITLE
ubo logger

### DISCUSSION THREAD

**Discussion by ghajini** [uBlockOrigin/uBlock-issues#2534] [Q&A] [answered]
UBO 1.47.5b5
chrome/firefox
now ubo logger doesn't show from which list a filter is invalid
![image](https://user-images.githubusercontent.com/20338483/224258746-26929996-d93d-4c72-83ac-812351a7c225.png)

**Comment by ghost** [comment]
From the one requested line below?

**Reply by ghajini** [reply]
those marked red ,it should show from which filter list it comes (like it was in previous versions)

**Comment by mapx-** [comment, marked as answer]
something similar does happen in case of popups => no indication about the popup origin

**Reply by ghajini** [reply]
can  you explain this part ,STR?

**Reply by mapx-** [reply]
when I'll get another example ..

**Reply by mapx-** [reply]
Here we are:

- chrome, uBO 1.47.5b4
- add `zerion.cc#@#+js(no-setTimeout-if, appendChild)`
- go to `https://zerion.cc/serial/the-walking-dead-2010/s02e04/cherokee-rose?id=2zVU_pFVz`
(or pick any other episode, choose `embedo`)
- click play

in the logger:
![image](https://user-images.githubusercontent.com/10303732/224272101-e39b09ad-1f51-43a9-a6ff-1ea6b092bb88.png)

**Reply by gorhill** [reply]
The popup is opened from within an `about:blank` frame, and it's not possible to extract a context hostname from `about:blank`. So either the logger reports `about-scheme`, or the owner of the anonymous frame, which is `zerion.cc` in that case.

**Reply by mapx-** [reply]
I guess reporting the owner should be ok

**Reply by gorhill** [reply]
Right, any filter will have to use the actual owner of the frame as context anyways.

**Comment by gorhill** [comment]
So there were two different unrelated issues reported here. These should be fixed in 1.47.5b6.
